subsys/logging/Kconfig

```diff
@@ -279,6 +279,40 @@ config LOG_FRONTEND
 	  When enabled, logs are redirected to a custom frontend instead
 	  of being processed by the logger.
 
+config LOG_BACKEND_FLASH
+	bool "Enable FLASH backend"
+	depends on FLASH
+	depends on FILE_SYSTEM_LITTLEFS
+	default n
+	help
+	  When enabled backend is using FLASH to output logs.
+
+if LOG_BACKEND_FLASH
+
+config OVERWRITE_OLD_LOG_FILES
+	bool "Enable old log files overwrite"
+	default y
+	help
+	  When enabled backend overwrites oldest log files.
+	  In other case, when memory is full, new messages are dropped.
+
+config LOG_FLASH_FILE_SIZE
+	int "Log file size as multiplication of flash page size."
```

Comment:
is it possible that file system is build on medium without pages (ram?) or backend relies on file system and flash api?